bin/shell.js

```diff
@@ -0,0 +1 @@
+require('../lib/shell.js')();
```

Comment:
How about making it executable instead, using `!#/usr/bin/env node` to start it?